pyquil/api/_compiler.py

```diff
@@ -13,11 +13,15 @@
 #    See the License for the specific language governing permissions and
 #    limitations under the License.
 ##############################################################################
-from typing import Dict, Optional
+from typing import Any, Dict, Optional
 
 from qcs_sdk import QCSClient
 from qcs_sdk.qpu.rewrite_arithmetic import rewrite_arithmetic
-from qcs_sdk.qpu.translation import get_quilt_calibrations, translate
+from qcs_sdk.qpu.translation import (
+    get_quilt_calibrations,
+    translate,
+    TranslationOptions as QPUCompilerAPIOptions,
```

Comment:
Thoughts about this rename? The original name makes sense where it's defined, but it makes less sense in pyQuil

Reply:
I think this is a good idea 👍